src/build/optimize-streams.ts

```diff
@@ -104,10 +104,24 @@ export class GenericOptimizeTransform extends Transform {
 class JSBabelTransform extends GenericOptimizeTransform {
   constructor(config: BabelTransformOptions) {
     const transform = (contents: string, options: BabelTransformOptions) => {
-      return babelTransform(contents, options).code!;
+      const es5Code = babelTransform(contents, options).code!;
+      return this._replaceTemplateObjectNames(es5Code);
     };
     super('.js', transform, config);
   }
+
+  /**
+   * Transforms given Babel-transpiled code so that the variable names
+   * of template literals are each unique
+   * @param code Transpiled code to evaluate
+   * @returns The code with all `_templateObject` variable
+   * names modified to be unique.
+   */
+  _replaceTemplateObjectNames(code: string): string {
+    // e.g., _templateObject -> _templateObject_200817b1154811e887be8b38cea68555
+    const uniqueId = uuid().replace(/-/g, '');
+    return code.replace(/_templateObject(?!_[A-Fa-f0-9]+)/g, `_templateObject_${uniqueId}`);
```

Comment:
not that it makes any functional difference but you could probably just check a word boundary here (`_templateObject\b`)

Reply:
Good call. Done